setup.py

```diff
@@ -17,6 +17,19 @@
 with open(os.path.join(here, 'sewer', '__version__.py'), 'r') as f:
     exec(f.read(), about)
 
+dns_provider_deps_map = {
+    'aliyun': ['aliyun-python-sdk-core-v3', 'aliyun-python-sdk-alidns'],
+    'alicloud': ['aliyun-python-sdk-core-v3', 'aliyun-python-sdk-alidns'],
```

Comment:
@menduo I think we should probably also just use either `aliyun` or `alicloud` rather than have them as different. 
But I'm not sure, I've never worked with aliyun or alicloud so I do not know how different they are. 
I'll leave you to make that judgement